### PR TITLE
Fixes #9672: Add Rudder high level diagram on Readme

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/9672

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -1,5 +1,8 @@
-Rudder: Continuous configuration for effective compliance
----------------------------------------------------------
+
+image::readme-resources/color_logo_horizontal_small.png[Rudder]
```
Comment (amousset) [inline README.asciidoc]: This looks great!

**Comment by amousset** [comment]
Some comments about the diagram:
* Some untranslateud words: "constructeur", "ID Rudder" -> "Rudder ID", "Authentification" -> "Authentication"
* "Softwares" -> "Software"
* "Network::" -> "Network:"
* "interfaces,, IP" -> "interfaces, IP"
* "Policies defined byt a set"

**Comment by amousset** [comment]
Also the colors good probably be better, but it's good to have this! We should add it to the manual too!

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
PR rebased

**Pull request merged**